docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -1,204 +1,309 @@
-commands, assigning `PROXY_ADDR` to the address of your Teleport Cloud tenant.
-
-```code
-$ PROXY_ADDR=<Var name="mytenant.teleport.sh" />:443
-
-# Install Kubernetes agent. It dials back to the Teleport cluster at $PROXY_ADDR
-$ CLUSTER=<Var name="cookie" />
-# Create the values.yaml file
-$ cat > values.yaml << EOF
-authToken: "${TOKEN}"
-proxyAddr: "${PROXY_ADDR}"
-roles: "kube,app,discovery"
-joinParams:
-  method: "token"
-  tokenName: "${TOKEN}"
-kubeClusterName: "${CLUSTER}"
-enterprise: true
-EOF
-# Run the helm install specifying to match to the Teleport Cloud version of Teleport
-$ helm install teleport-agent teleport/teleport-kube-agent \
-  -f values.yaml \
-  --create-namespace \
-  --namespace=teleport-agent \
-  --version (=cloud.version=)
-```
-
-</TabItem>
-</Tabs>
-
-Make sure that the Teleport agent pod is running. You should see one Teleport
-agent pod pod with a single ready container:
-
-```code
-$ kubectl -n teleport-agent get pods
-NAME               READY   STATUS    RESTARTS   AGE
-teleport-agent-0   1/1     Running   0          32s
-```
-
-## Step 3/3 Access your Kubernetes cluster
-
-(!docs/pages/includes/kubernetes-access/rbac.mdx!)
-
-### View pods in your cluster
-
-List connected clusters using `tsh kube ls` and switch between
-them using `tsh kube login`:
-
-```code
-$ tsh kube ls
-
-# Kube Cluster Name Selected
-# ----------------- --------
-# cookie
-
-# kubeconfig now points to the cookie cluster
-$ tsh kube login cookie
-# Logged into kubernetes cluster "cookie". Try 'kubectl version' to test the connection.
-
-# kubectl command executed on `cookie` but is routed through the Teleport cluster.
-$ kubectl get pods
-```
-
-## Next Steps
-
-- Take a look at a [kube-agent helm chart reference](../reference/helm-reference/teleport-kube-agent.mdx) for a full list of parameters.
+## Step 1/4. Select guided or manual enrollment
+
+There are two options for enrolling a Kubernetes cluster as a resource:
+
+- You can follow the guided enrollment steps in the Teleport Web UI.
+- You run commands and edit files manually in a terminal.
+
+The guided enrollment simplifies the deployment process by pre-populating commands
+and files with required information—for example, the token used to provision the
+Kubernetes cluster—and information you specify, such as the Kubernetes namespace,
+users, and groups to grant access to.
+
+If you can't use the Teleport Web UI or want to run commands and edit file manually,
+see [Enroll Kubernetes manually](#enroll-kubernetes-manually).
+
+## Step 2/4. Follow guided enrollment
+
+To enroll a Kubernetes cluster using the Teleport Web UI:
+
+1. Open the Teleport Web UI and sign in using your administrative account.
+
+1. Click **Enroll New Resources**.
+
+1. Type all or part of **Kubernetes** in the Search field to filter the resource types 
+   displayed, then click **Kubernetes**.
+
+1. Copy the command to add the `teleport-agent` chart to your charts repository and paste 
```

Comment:
Is this the `teleport-kube-agent` chart? https://goteleport.com/docs/reference/helm-reference/teleport-kube-agent/

Reply:
From the UI, it says:
Add teleport-agent chart to your charts repository

I'm not sure, but I think teleport-kube-agent is an example chart, but I'm not sure it's exactly what gets added here (I'm guessing that this is a boilerplate that your settings in the UI fill in, not the teleport-kube-agent chart as defined in the reference section.)

Reply:
It looks like the command shown in the [source](https://github.com/gravitational/teleport/blob/8c41503fd4e24b14bf435b6fc55055d76549b5c0/web/packages/teleport/src/Discover/Kubernetes/HelmChart/HelmChart.tsx#L359), `helm install teleport-agent teleport/teleport-kube-agent`, installs the `teleport-kube-agent` chart but names the [release](https://helm.sh/docs/intro/using_helm/) `teleport-agent`.